src/main/java/org/broadinstitute/hellbender/transformers/BaseQualityClipReadTransformer.java

```diff
@@ -29,8 +30,8 @@ public BaseQualityClipReadTransformer(int trim_thresh) {
      *
      */
     @Override
-    public GATKRead apply(GATKRead read) {
-        GATKRead readClippedRightEnd = clipReadRightEnd(read);
+    public GATKRead apply(final GATKRead read) {
+        final GATKRead readClippedRightEnd = clipReadRightEnd(read);
         return clipReadLeftEnd(readClippedRightEnd);
     }
 
```

Comment:
That was my original approach, but I think it's easier to have two simple functions instead of one complicated function in this case.